src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+import "./Introduce.scss";
+
+function Introduce() {
+  function handleInput() {
+    console.log("input 태그 동작");
+  }
+
+  return (
+    <div className="wecode">
+      <div className="Wrap">
```

Comment:
```suggestion
      <div className="wrap">
```
첫 글자를 대문자로 하는 것은 컴포넌트의 이름을 지을 때 사용하는 컨벤션입니다.
className은 다른 case를 적용해주세요!